### PR TITLE
Add referrals table v2

### PR DESCRIPTION
**Pull request by kelvin-chappell**

### CODE REVIEW

**Comment by kelvin-chappell** [comment]
This is to take data written in https://github.com/guardian/ophan-data-lake/pull/1104

File: cloudformation/dynamoDb.yaml [hunk 1]
```diff
@@ -220,6 +220,30 @@ Resources:
           - referrals
     DeletionPolicy: Retain
 
+  CampaignReferralsV2DBTable:
```
Comment (LATaylor-guardian) [inline cloudformation/dynamoDb.yaml:223]: Is there a way we can name this so it's agnostic of the temporary api versioning we put in place (as I plan to remove this at a later date)?

Comment (kelvin-chappell) [reply cloudformation/dynamoDb.yaml:223]: This isn't linked to the api versioning.  It's just that the data structure has changed and I don't want to break the app.  Will remove versioned table names when data is stable and just use original referrals table.

Comment (LATaylor-guardian) [reply cloudformation/dynamoDb.yaml:223]: Got it. In that case 👍

**Comment by kelvin-chappell** [comment]
[Reporting: Referrals: Impressions and CTR per card](https://trello.com/c/SjjlWMKh/146-reporting-referrals-impressions-and-ctr-per-card)

**Pull request merged**